.snippets/code/tutorials/processor.ts

```diff
@@ -0,0 +1,47 @@
+import {assertNotNull} from '@subsquid/util-internal'
+import {lookupArchive} from '@subsquid/archive-registry'
+import {
+    BlockHeader,
+    DataHandlerContext,
+    EvmBatchProcessor,
+    EvmBatchProcessorFields,
+    Log as _Log,
+    Transaction as _Transaction,
+} from '@subsquid/evm-processor'
+import {Store} from '@subsquid/typeorm-store'
```

Comment:
store doesn't seem to be used either

Reply:
store is needed, e.g. `await ctx.store.upsert(Array.from(accounts.values()))`